README.md

```diff
@@ -119,6 +119,7 @@ If you would like to customize how fields get added to your form, following opti
     affiliate: "IAFFILIATE"
   }; // Maps the initial affiliate parameter to IAFFILIATE form field
 
+  _uf.track_last_params          = true; // Enable the tracking of last parameters even if null
```

Comment:
Perhaps using `track_null_params` with comment _**Default: false. Use empty values if UTM not present on the same page as the form**_ can clarify this.

Reply:
Agree.

Reply:
Rethinking about this (based on your `current_session` changes), it seems that the correct setting might be `reset_params_at_session_end`. 

Also, the comment could be _**Default: false. Use empty string values if UTM not present in the same session as the form submit**_